src/lib/corchuelo.rs

```diff
@@ -88,7 +88,7 @@ pub(crate) fn recover<TokId: Clone + Copy + Debug + TryFrom<usize> + TryInto<usi
             },
             _ => {
                 let num_inserts = repairs.vals()
-                                         .filter(|r| if let ParseRepair::InsertTerm{..} = **r {
+                                         .filter(|r| if let ParseRepair::Insert{..} = **r {
```

Comment:
Why did you rename these in the second commit, only to rename them back here?

Reply:
Hmm, that's a rebasing gone wrong (my fault). I can fix things if you want before you do a full review?

Reply:
I'll try to finish my review first.

Reply:
I'm surprised this didn't result in an error.

Reply:
It's not an error-error, just a mistake in the rebasing on my part I think.

Reply:
Just saw, this get's refixed in the next commit which is why the tests don't catch this.